readthedocs/core/tasks.py

```diff
@@ -0,0 +1,47 @@
+"""Basic tasks"""
+
+import logging
+
+from celery import task
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import get_template
+
+
+log = logging.getLogger(__name__)
+
+
+@task(queue='web')
+def send_email_task(recipient, subject, template, template_html, context=None):
+    """
+    Send multipart email
+
+    recipient
+        Email recipient address
+
+    subject
+        Email subject header
+
+    template
+        Plain text template to send
+
+    template_html
+        HTML template to send as new message part
+
+    context
+        A dictionary to pass into the template calls
+
+    request
+        Request object for determining absolute URL
+    """
+    ctx = {}
+    ctx.update(context)
```

Comment:
Doesn't seem so, unless `context` needs to be coerced to a dict. I don't see this being an issue though.